client/src/components/dashboard/Inbox.js

```diff
@@ -24,51 +25,54 @@ function Inbox() {
     getMessages();
   }, []);
 
+  //delete message
+
   async function deleteMessage(id) {
     try {
-      await fetch(`/messages/${id}`, {
+      await fetch(`/dashboard/messages/${id}`, {
         method: "DELETE",
-        headers: { "Content-Type": "application/json" },
+        headers: { jwt_token: localStorage.token },
       });
+
       setMessages(messages.filter((message) => message.message_id !== id));
     } catch (err) {
       console.error(err.message);
     }
   }
+
   useEffect(() => {
     setMessages(allMessages);
   }, [allMessages]);
-  console.log(allMessages);
-  console.log(messages);
-  console.log(messages.length, messages[0]);
+  // console.log(allMessages);
+  // console.log(messages);
+  // console.log(messages.length, messages[0]);
 
   return (
     <>
       <section>
         <div className="icon-heading">
-          <h1>Inbox</h1>
+          <h2>Inbox</h2>
           <img alt="messages icon" src={messagesIcon} />
         </div>
-
-        <section>
-          {messages.length !== 0 &&
-            messages[0].message_id !== null &&
-            messages.map((message) => (
-              <div>
-                <div key={message.message_id}>{message.message_text}</div>
-                <button onClick={() => deleteMessage(message.message_id)}>
-                  Delete
-                </button>
-              </div>
-            ))}
-        </section>
-
         <div className="login-signin-buttons">
           <Link to="/new-message">
```

Comment:
Link/button issue here as well (btw I recognise this was in the previous code, but as it's showing up in the changelist I may as well catch it).